### PR TITLE
add convertor test

### PR DESCRIPTION
**Pull request by shivupa**
This adds a test

### CODE REVIEW

File: tests/converter/test_LiH_MSD_pyscf/cmd_args.txt
Comment (amandadumi) [inline tests/converter/test_LiH_MSD_pyscf/cmd_args.txt:1]: Does this empty file need to be here?

File: tests/converter/test_LiH_MSD_pyscf/gold.wfnoj.xml
Comment (amandadumi) [inline tests/converter/test_LiH_MSD_pyscf/gold.wfnoj.xml:1]: why are these named gold?

**Review by amandadumi — CHANGES_REQUESTED** [review]
This is perfect, thanks for adding it. Just a few questions. If you don't have bandwidith i can just approve and remove them myself, but wanted to be sure they weren't intentional.

File: tests/converter/test_LiH_MSD_pyscf/gold.wfnoj.xml
Comment (shivupa) [reply tests/converter/test_LiH_MSD_pyscf/gold.wfnoj.xml:1]: idk it seemed like all the reference values are called gold? I guess like gold standard? Sorta confusing imo

File: tests/converter/test_LiH_MSD_pyscf/cmd_args.txt
Comment (shivupa) [reply tests/converter/test_LiH_MSD_pyscf/cmd_args.txt:1]: not sure but its in the other pyscf test lol

**Comment by amandadumi** [comment]
looks good thank you!

**Pull request merged**